### DISCUSSION TITLE
Turn on the printer with PSU Control when waking Octodash

### DISCUSSION THREAD

**Discussion by StreckerCM** [UnchartedBull/OctoDash#1382] [Q&A] [answered]
I have my Octodash setup and a custom action to turn off the PSU but to turn it on I have to go into the printer web client and hit the little lightning bolt to get it to switch the printer on.  Is it possible to tell the PSU control plugin to power on the printer when waking Octodash?

**Comment by UnchartedBull** [comment, marked as answer]
There is an option in the settings under the PSUPlugin called "Turn on when exiting sleep" or similar. This does exactly what you describe.

**Reply by fizzyade** [reply]
@UnchartedBull I don't have this option in PSUControl, any other suggestions?

![CleanShot 2021-07-06 at 08 55 33@2x](https://user-images.githubusercontent.com/55795671/124563563-0ea28980-de38-11eb-8ffa-382bc527ca8d.png)

**Reply by jneilliii** [reply]
Because it's an octodash setting...

**Reply by fizzyade** [reply]
And it’s not in the Octodash settings either in the GUI.

I managed to find out how to do it in the end, by editing the config.json, but it seems the option was removed from the GUI sometime in recent history.

**Reply by morgendagen** [reply]
In case anyone else (my future self included) stumbles upon this issue: edit `~/.config/octodash/config.json`, search for `psuControl` and set `turnOnPSUWhenExitingSleep` to `true`. Also set `turnOnPrinterWhenExitingSleep` to `true`.

**Reply by atiniir** [reply]
@morgendagen thank you! appreciate the clarity.
